wavefront_pyformance/runtime_metrics.py

```diff
@@ -115,7 +115,11 @@ def collect_processes(self):
 
     def collect_contextswitches(self):
         """Collect Context Switches."""
-        ctx_switches = self.process.num_ctx_switches()
+        try:
+            ctx_switches = self.process.num_ctx_switches()
+        except NotImplementedError:  # some platforms do not expose ctx switches information, this is okay
```

Comment:
The line would be too long to pass the linters, please move the comment to it's own line

Reply:
Done